libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -135,8 +135,8 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
               // Just a failsafe in case this.isLoading for some reason is not reset
               if (elapsedTime > TIMEOUT) {
```

Comment:
If I understand correctly, we stop loading after the TIMEOUT has elapsed & the user has to scroll down again to trigger the load. If that is the case, should consumers of the list, perhaps be able to set the timeout themselves, in case they have some large queries?

Reply:
Actually, if the timeout is triggered, an empty set is returned - which means that the virtual list will stop querying more items entirely (so it may be a good idea to have the option to increase it)